Extending-OSVR/ConfiguringHDKViveTracking.md

```diff
@@ -0,0 +1,260 @@
+### OSVR-Vive device descriptor (com_osvr_Vive.json):
+````json
+{
+    "deviceVendor": "HTC",
+    "deviceName": "Vive PRE and Vive Controllers",
+    "author": "Georgiy Frolov <gfrolov@sensics.com>",
+    "version": 1,
+    "lastModified": "",
+    "interfaces": {
+        "tracker": {
+            "count": 4,
+            "bounded": true,
+            "position": true,
+            "orientation": true
+        },
+        "analog": {
+            "count": 7
+        },
+        "button": {
+            "count": 14
+        }
+    },
+    "semantic": {
+		"hmd": {
+            "$target": "tracker/0",
+			"button": "button/0",
+            "proximity": "button/1"
+		},
+		"puck": {
+            "$target": "tracker/3"
+		},
+        "ipd": "analog/0",
+        "controller": {
+            "left": {
+                "$target": "tracker/1",
+                "system": "button/2",
+                "menu": "button/3",
+                "grip": "button/4",
+                "trackpad": {
+                    "x": "analog/1",
+                    "y": "analog/2",
+                    "touch": "button/5",
+                    "button": "button/6"
+                },
+                "trigger": {
+                    "$target": "analog/3",
+                    "button": "button/7"
+                }
+            },
+            "right": {
+                "$target": "tracker/2",
+                "system": "button/8",
+                "menu": "button/9",
+                "grip": "button/10",
+                "trackpad": {
+                    "x": "analog/4",
+                    "y": "analog/5",
+                    "touch": "button/11",
+                    "button": "button/12"
+                },
+                "trigger": {
+                    "$target": "analog/6",
+                    "button": "button/13"
+                }
+            }
+        }
+    },
+    "automaticAliases": {
+        "/me/head": "semantic/hmd",
+        "/me/puck": "semantic/puck",
+        "/me/ipd": "semantic/ipd",
+        "/controller": "semantic/controller/*",
+        "/me/hands/left": "semantic/controller/left",
+        "/me/hands/right": "semantic/controller/right"
+    }
+}
+````
+
+Take the time to review the device descriptor to understand how the Vive devices are factored into OSVR interfaces. The parts related to the puck are the assignment to tracker 3, and the automatic alias:
+````json
+"/me/puck": "semantic/puck",
+````
+
+### OSVR Server Configuration
+Normally for HDK2 with orientation-only tracking, we’d use a config file such as [/sample-configs/osvr_server_config.renderManager.HDKv2.0.direct.json](https://github.com/OSVR/OSVR-Core/blob/master/apps/sample-configs/osvr_server_config.renderManager.HDKv2.0.direct.json). 
```

Comment:
I'd spell out the words as the docs are a little more formal (_e.g._, HDK2 → OSVR HDK 2, config → configuration).